### PR TITLE
Add profile README

### PR DESCRIPTION
**Pull request by maxjacobson**
Rendered: https://github.com/dolfamli/.github/blob/mj/add-profile-readme/profile/README.md

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Maryland Department of Labor (DOL)
```
Comment (angeliquejw) [inline profile/README.md]:
```suggestion
# Maryland Department of Labor (MDOL)
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+## Family and Medical Leave Insurance (FAMLI)
+
+This is the GitHub organization for the digital services team of [FAMLI](https://paidleave.maryland.gov/Pages/default.aspx).
```
Comment (angeliquejw) [inline profile/README.md]:
```suggestion
This is the GitHub organization for the digital services team of [FAMLI](https://paidleave.maryland.gov/).
```

**Review by angeliquejw — APPROVED** [review]
2 small tweaks here, but this gets the job done 👍🏻

**Comment by maxjacobson** [comment]
> 2 small tweaks here, but this gets the job done 👍🏻

Thanks!

Heh yeah, a very bare bones starting point. Something to iterate on :)

**Pull request merged**